menas/src/main/scala/za/co/absa/enceladus/menas/HDFSConfig.scala

```diff
@@ -66,20 +62,32 @@ class HDFSConfig @Autowired()(spark: SparkSession) {
 
       logger.info("Using kerberos hadoop authentication")
 
-      System.setProperty("sun.security.krb5.debug", krb5debug);
-      System.setProperty("java.security.krb5.realm", krb5realm);
-      System.setProperty("java.security.krb5.kdc", krb5kdc);
+      System.setProperty("sun.security.krb5.debug", krb5debug)
+      System.setProperty("java.security.krb5.realm", krb5realm)
+      System.setProperty("java.security.krb5.kdc", krb5kdc)
 
 
       val c = hadoopConf()
-      
+
       SecurityUtil.setAuthenticationMethod(AuthenticationMethod.KERBEROS, c)
       UserGroupInformation.reset()
-      UserGroupInformation.setConfiguration(c);
+      UserGroupInformation.setConfiguration(c)
       UserGroupInformation.loginUserFromKeytab(krb5username, krb5keytab)
 
       FileSystem.get(c)
-      
-    } else FileSystem.get(hadoopConf)
+
+    } else if (authMethod == "simple") {
+      logger.info(s"Using simple hadoop authentication as $authUser")
+      val conf = hadoopConf()
+      conf.set("hadoop.security.authentication", "simple")
+      SecurityUtil.setAuthenticationMethod(AuthenticationMethod.SIMPLE, conf)
```

Comment:
Aren't those two lines doing the same thing?

Reply:
It is very likely - will remove the first one